### PR TITLE
feat: Added Data Converter in Kitchen Sink

### LINKED ISSUE #764: Kitchen Sink: Data Converter
### Describe the feature

I would like to implement the Discount Calculator feature!

- This app will convert the data value based on user input such as MB to GB etc.

Hey @tobySolutions, Please assign this issue to me under hactoberfest label. Thank you!

### Additional information

- [X] Would you be willing to help implement this feature?

### PR DESCRIPTION
**Pull request by lalitkumawat1m**
**Please describe the changes this PR makes and why it should be merged:**
Added Data Converter in Kitchen Sink

Resolves #764

**Status**

- [x] Code changes have been tested against prettier, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [ ] This PR changes the codebase
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
  - [ ] This PR changes the internal workings with no modifications to the external API (bug fixes, performance improvements)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Review by tobySolutions — APPROVED** [review]
LGTM!! 🙌

File: packages/kitchen-sink/src/examples/data-converter.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+enum DataUnit {
+Byte = "Byte",
+Kilobyte = "KB",
+Megabyte = "MB",
+Gigabyte = "GB",
+Terabyte = "TB",
+Petabyte = "PB",
+}
+
+const DataConverter: React.FC = () => {
```
Comment (tobySolutions) [inline packages/kitchen-sink/src/examples/data-converter.tsx:19]: Your component doesn't use Million though. Can you maybe check this out?  Use the `block` function from Millon

**Review by tobySolutions — CHANGES_REQUESTED** [review]

**Comment by lalitkumawat1m** [comment]
Hey @tobySolutions, I have added a block function. Could you please check now?

**Review by tobySolutions — APPROVED** [review]
LGTM!

**Comment by tobySolutions** [comment]
Good work @lalitkumawat1m!

**Pull request merged**